### PR TITLE
Documentation and example for ToolButton

### PR DESCRIPTION
**Pull request by Hrishi1999**
Here are the final changes. The example is working, and the documentation is as it should be.

### CODE REVIEW

File: examples/toolbutton.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+toolbar_box = ToolbarBox()
+vbox.pack_start(toolbar_box, False, False, 0)
+toolbar_box.show()
+
+separator = Gtk.SeparatorToolItem()
+toolbar_box.toolbar.insert(separator, -1)
+separator.show()
+
+tool_button = ToolButton(icon_name='view-radial')
```
Comment (i5o) [inline examples/toolbutton.py]: Maybe you could add a tooltip to the button?

**Comment by Hrishi1999** [comment]
@i5o , yeah I can. But I am having problem squashing commits, so with every change, I think i will need to remove my repo, fork it again and submit my changes with a new pr. So, let's wait for quozl, then i will do all that changes at once, to avoid creating more prs.

**Comment by Hrishi1999** [comment]
Done @i5o @quozl @samdroid-apps @walterbender

**Comment by quozl** [comment]
Thanks for your work on this, and as I understand you have run out of time to squash commits or make further changes, I've merged your work with mine and others in #349.

**Comment by i5o** [comment]
Replaced by #349 // also merged, thanks!

Good job @Hrishi1999 and @quozl

**Pull request closed without merging**